wire-ios-data-model/Source/ManagedObjectContext/Migration/CoreDataMessagingMigrationVersion.swift

```diff
@@ -65,8 +66,10 @@ enum CoreDataMessagingMigrationVersion: String, CaseIterable {
 
     var nextVersion: Self? {
         switch self {
-        case .version2_112:
+        case .version2_113:
             return nil
+        case .version2_112:
+            return .version2_113
         case .version2_111:
             return .version2_112
```

Comment:
Actually after reflecting on this I think we can safely change it this way:

```swift
        case .version2_113:
            return nil
        case .version2_112:
            return .version2_113 // heavy weight migration
        case .version2_107,
                .version2_108,
                .version2_109,
                .version2_110,
                .version2_111:
            return .version2_112
        case .version2_106:
            return .version2_107 // heavy weight migration
        case .version2_80,
```

Why? because we let CoreData automatically interfere the mapping model for light weight migrations between the current model and next model. This way we save quite a few migration steps, that actually should make the migration faster.
`NSMappingModel.inferredMappingModel(forSourceModel: sourceModel, destinationModel: destinationModel)`.

What do you think? We could also do it on a separate branch if you don't want to do it here.

Reply:
I'm not sure I understand fully the effects of this change. But I think it would make sense to break out a new ticket work on it separately.

Reply:
okay, will do so.